aioairq/core.py

```diff
@@ -9,6 +9,10 @@
 from aioairq.exceptions import InvalidAirQResponse, InvalidIpAddress
 from aioairq.utils import is_valid_ipv4_address
 
+LedThemeName = Literal[
+    "standard", "co2_covid19", "CO2", "VOC", "CO", "PM1", "PM2.5", "PM10", "Noise"
```

Comment:
This list is not complete.
For example
```py
await airq.set_led_theme({"right": "Noise (contrast)"})
```
will be criticized by my IDE, however, it's totally fine.